docs/features/techdocs/getting-started.md

```diff
@@ -110,21 +252,11 @@ Python packages.
 
 ## Run Backstage locally
```

Comment:
Not part of PR, but consider renaming it "Running Backstage locally" to conform to the style used in the other headings.